Home_04/task8.py

```diff
@@ -0,0 +1,19 @@
+#   Задача 8:
+#   Требуется определить, можно ли от шоколадки 
+#   размером n × m долек отломить k долек, если разрешается сделать один
+#   разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+#   *Пример:*
+#   3 2 4 -> yes
+#   3 2 1 -> no
+
+n,m,k = map(int,input().split())
+if n * m <= k:
+    print("Шоколадка не делится")
+elif n == m and k % n == 0:
+    print("Можно делить шоколадку")
+elif n == m and k % n != 0:
+     print("Шоколадка не делится")
+elif n != m and k % n == 0 or k % m == 0:
```

Comment:
Достаточно только части последнего условия.
```suggestion
elif k % n == 0 or k % m == 0:
```